public/nginx.conf

```diff
@@ -62,23 +68,33 @@ http {
             add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;
             add_header X-Frame-Options SAMEORIGIN always;
             add_header X-Content-Type-Options nosniff always;
-            add_header Referrer-Policy no-referrer-when-downgrade always;
+            add_header Referrer-Policy strict-origin-when-cross-origin always;
             add_header Permissions-Policy "geolocation=(), microphone=(), camera=() always";
             add_header Content-Security-Policy "
-                default-src 'self' https:;
-                script-src 'self' 'unsafe-inline' 'unsafe-eval' https: https://cdn.logrocket.io https://cdn.lr-ingest.io https://cdn.lr-in.com https://cdn.lr-in-prod.com https://cdn.lr-ingest.com https://cdn.ingest-lr.com https://cdn.lr-intake.com https://cdn.intake-lr.com;
-                style-src 'self' 'unsafe-inline' https:;
-                img-src 'self' data: https:;
+                default-src 'none'
+
+                connect-src 'self' https: https://*.logrocket.io https://*.lr-ingest.io https://*.logrocket.com https://*.lr-in.com https://*.lr-in-prod.com https://*.lr-ingest.com https://*.ingest-lr.com https://*.lr-intake.com https://*.intake-lr.com https://*.logr-ingest.com https://api.stripe.com https://maps.googleapis.com;
+                
                 font-src 'self' data: https:;
-                connect-src 'self' https: https://*.logrocket.io https://*.lr-ingest.io https://*.logrocket.com https://*.lr-in.com https://*.lr-in-prod.com https://*.lr-ingest.com https://*.ingest-lr.com https://*.lr-intake.com https://*.intake-lr.com https://*.logr-ingest.com;
-                object-src 'none';
+                
+                frame-src 'self' https://*.estuary.dev https://*.js.stripe.com https://js.stripe.com https://hooks.stripe.com https://td.doubleclick.net;
+                
+                img-src 'self' data: https:;
+                
+                script-src 'self' 'unsafe-eval' https:*estuary.dev https://cdn.logrocket.io https://cdn.lr-ingest.io https://cdn.lr-in.com https://cdn.lr-in-prod.com https://cdn.lr-ingest.com https://cdn.ingest-lr.com https://cdn.lr-intake.com https://cdn.intake-lr.com https://*.google-analytics.com https://*.analytics.google.com https://*.googletagmanager.com https://*.g.doubleclick.net https://*.google.com https://*.reo.dev https://*.stripe.network https://*.js.stripe.com https://js.stripe.com https://maps.googleapis.com;
```

Comment:
Removed `unsafe-inline` and `https:` and then added in our own domain, GTM, Stripe, and Reo.